src/Schema/Events/Tap.ts

```diff
@@ -327,6 +327,29 @@ export interface TappedFairGroup extends TappedEntityGroup {
   action: ActionType.tappedFairGroup
 }
 
+/**
+ * A user taps a grouped card (Explore By, Discover Something new)
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCardGroup",
+ *    context_module: "exploreBySection" | discoverSomethingNewSection,
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_type: "collectionsCategory" | "collection",
+ *    destination_screen_owner_id: "5359794d1a1e86c3740001f7" | undefined,
```

Comment:
**suggestion**: Even though technically this can be undefined as the type allows, I'd specify just the string value here in the example.